logstash-filter-truncate.gemspec

```diff
@@ -0,0 +1,26 @@
+Gem::Specification.new do |s|
+
+  s.name            = 'logstash-filter-truncate'
+  s.version         = '0.0.1'
```

Comment:
I try to start new projects at 0.0.1 as a practice.

I'm fine with 1.0.0, though, an dwill make that change.